### PR TITLE
Lower accuracy assertion for ResNet50.

### PR DESCRIPTION
**Pull request by will-cromar**

### CODE REVIEW

File: tests/pytorch/nightly/resnet50-mp.libsonnet [hunk 1]
```diff
@@ -65,7 +65,7 @@ local tpus = import 'templates/tpus.libsonnet';
               FINAL: {
                 fixed_value: {
                   comparison: 'GREATER',
-                  value: 75.0,
+                  value: 74.0,
```
Comment (JackCaoG) [inline tests/pytorch/nightly/resnet50-mp.libsonnet:68]: Does this change both pod and donut?

Comment (will-cromar) [reply tests/pytorch/nightly/resnet50-mp.libsonnet:68]: Yes. Do we want to keep it at 75% for donut?

Comment (JackCaoG) [reply tests/pytorch/nightly/resnet50-mp.libsonnet:68]: ideally yes.. This way when donut regress we can catch that.

Comment (will-cromar) [reply tests/pytorch/nightly/resnet50-mp.libsonnet]: Done. Checked the diff and this only affects the pod version of the test.

**Review by JackCaoG — APPROVED** [review]

**Review by wonjoolee95 — APPROVED** [review]

**Pull request merged**